diffblue.yml

```diff
@@ -5,20 +5,61 @@ phases:
  timeout: 300
```

Comment:
:grey_question: I can't comment on line 2, but @allredj didn't you say that you've seen cases where a high depth caused problems? Should we set it to lower values in phases 1 and 2? Or maybe that should be left for the fine-tuning and is out of scope of this PR.

Reply:
We seem to use `depth: false` on all our benchmarks at the moment, the platform default values are 1500, 3000 and 6000. I think this is an option to be fine-tuned later as it's not clear how much say an extra 1000 depth would affect things in a general case.

Reply:
`depth 1500` works in 95% of cases. I would use it in most phases. There are cases where we need higher values, like 3000. Maybe we could use that in fallback phases.

I've seen at least a few cases where `depth: false` is bad and prevents test-gen from terminating.

Reply:
Ok, I'll use 1500 in the first two phases, 3000 in third phase. I'm testing fallbacks that copy 1st and 3rd phase now so that means the higher depth would appear there too.